packages/micropip/micropip/micropip.py

```diff
@@ -103,40 +103,36 @@ def _parse_wheel_url(url: str) -> Tuple[str, Dict[str, Any], str]:
     return name, wheel, version
 
 
-class _WheelInstaller:
-    def extract_wheel(self, fd):
-        with zipfile.ZipFile(fd) as zf:
-            zf.extractall(WHEEL_BASE)
-
-    def validate_wheel(self, data, fileinfo):
-        if fileinfo.get("digests") is None:
-            # No checksums available, e.g. because installing
-            # from a different location than PyPi.
-            return
-        sha256 = fileinfo["digests"]["sha256"]
-        m = hashlib.sha256()
-        m.update(data.getvalue())
-        if m.hexdigest() != sha256:
-            raise ValueError("Contents don't match hash")
+def extract_wheel(fd):
```

Comment:
We probably should call all of these with a leading `_`, e.g. `_extract_wheel` to make it explicit that they are not part of the public API.